pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +213,16 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("persistentvolumes", "persistentvolumes/status", "storageclasses").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("events", "limitranges", "namespaces", "namespaces/status",
```

Comment:
@screeley44 what 'admin' creates the quotas then if the Storage admin doesn't have access to do this?